stestr/commands/list.py

```diff
@@ -94,12 +114,19 @@ def list_command(config='.stestr.conf', repo_type='file', repo_url=None,
     :param str group_regex: Set a group regex to use for grouping tests
         together in the stestr scheduler. If both this and the corresponding
         config file option are set this value will be used.
-    :param str blacklist_file: Path to a blacklist file, this file contains a
-        separate regex exclude on each newline.
-    :param str whitelist_file: Path to a whitelist file, this file contains a
-        separate regex on each newline.
-    :param str black_regex: Test rejection regex. If a test cases name matches
-        on re.search() operation, it will be removed from the final test list.
+    :param str blacklist_file: DEPRECATED: soon to be replaced by the new
+        option exclusion_list_file below.
+    :param str exclusion_list_file: Path to an exclusion list file, this file
+        contains a separate regex exclude on each newline.
+    :param str whitelist_file: DEPRECATED: soon to be replaced by the new
+        option inclusion_list_file below.
+    :param str inclusion_list_file: Path to an inclusion list file, this file
+        contains a separate regex on each newline.
+    :param str black_regex: DEPRECATED: soon to be replaced by the new
```

Comment:
We should add a deprecation warning for all these new deprecated arguments. Something like:

```python3
if black_regex is not None:
    warnings.warn("The black_regex argument is deprecated and will be removed in a future "
                  "release. Instead you should use exclude-regex which is functionally equivalent",
                  DeprecationWarning)
```

Reply:
👍